plugins/in_tail/tail_dockermode.c

```diff
@@ -224,29 +238,67 @@ int flb_tail_dmode_process_content(time_t now,
                                    char* line, size_t line_len,
                                    char **repl_line, size_t *repl_line_len,
                                    struct flb_tail_file *file,
-                                   struct flb_tail_config *ctx)
+                                   struct flb_tail_config *ctx,
+                                   msgpack_sbuffer *mp_sbuf, msgpack_packer *mp_pck
+                                   )
 {
     char* val = NULL;
     size_t val_len;
     int ret;
+    void *out_buf = NULL;
+    size_t out_size;
+    struct flb_time out_time = {0};
+
+    if (flb_sds_len(file->dmode_lastline) > 0 && file->dmode_complete) {
+#ifdef FLB_HAVE_REGEX
```

Comment:
this macro conditional should cover the ```if``` statement, otherwise the if does nothing.